### PR TITLE
Added 08 Whats New Page + Image

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Added 08 Whats New Page + Image

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Review by teallarson — APPROVED** [review]

File: pages/whats-new/2022-01-28-v08.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  alt="Salesforce Destinations"
+  src="v08-release/model-centric-ui.png"
+  width={1000}
+  height={424}
+/>
+
+Additionally we have added a few more integrations:
+
+- Airtable
+- Rollbar
```
Comment (pedroslopez) [inline pages/whats-new/2022-01-28-v08.mdx]: Rollbar isn't a source/destination, but rather a monitoring tool - maybe it shouldn't be grouped in with these?

File: pages/whats-new/2022-01-28-v08.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Klaviyo
+- Vero
+
+And have improved how we interect with Google Sheets as a Source!
```
Comment (pedroslopez) [inline pages/whats-new/2022-01-28-v08.mdx]:
```suggestion
And have improved how we interact with Google Sheets as a Source!
```

File: pages/whats-new/2022-01-28-v08.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+There are some breaking changes in this release so please see [this](https://www.grouparoo.com/docs/support/upgrading-grouparoo/v07-v08#migration-notes-and-breaking-changes) page for details and instructions.
```
Comment (pedroslopez) [inline pages/whats-new/2022-01-28-v08.mdx]:
```suggestion
There are some breaking changes in this release so please see [this](https://www.grouparoo.com/docs/support/upgrading-grouparoo/v07-v08#migration-notes-and-breaking-changes) page for details and instructions on how to upgrade.
```

File: pages/whats-new/2022-01-28-v08.mdx (hunk 1 above)
Comment (krishnaglick) [reply pages/whats-new/2022-01-28-v08.mdx]: What are we doing with it then?

Comment (pedroslopez) [reply pages/whats-new/2022-01-28-v08.mdx]: We have a few of these types of plugins (Sentry, Rollbar, New Relic...) that don't provide sources or destinations, but are used for sending metrics / reporting errors from the Grouparoo app itself to these various services. It's kind of its own category of plugins.

File: pages/whats-new/2022-01-28-v08.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+title: "Grouparoo v0.8 adds a Model-centric UI and more!"
+date: "2022-01-28"
+tags: ["new"]
+description: "Grouparoo v0.8 adds the ability to integrate with Airtable, Rollbar, Klaviyo, and Vero!"
+---
+
+The 0.8 release had a big focus on user experience with the addition of our Model-Centric UI!
+
+<Image
+  alt="Salesforce Destinations"
```
Comment (bleonard) [inline pages/whats-new/2022-01-28-v08.mdx]: This alt looks wrong.

File: pages/whats-new/2022-01-28-v08.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Grouparoo v0.8 adds a Model-centric UI and more!"
```
Comment (bleonard) [inline pages/whats-new/2022-01-28-v08.mdx]: Swtiching to the term `Model Overview UI`

**Review by bleonard — APPROVED** [review]

File: pages/whats-new/2022-01-28-v08.mdx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Grouparoo v0.8 adds a Model Overview UI and more!"
+date: "2022-02-03"
+tags: ["new"]
+description: "Grouparoo v0.8 adds the ability to integrate with Airtable, Klaviyo, and Vero!"
+---
+
+The 0.8 release had a big focus on user experience with the addition of our Model-Centric UI!
```
Comment (pedroslopez) [inline pages/whats-new/2022-01-28-v08.mdx]: This sentence still references "Model-Centric UI"

File: pages/whats-new/2022-01-28-v08.mdx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+We now support sending metrics to Rollbar.
+
+And added a few more integrations:
+
+- Airtable
+- Klaviyo
+- Vero
+
+And have improved how we interact with Google Sheets as a Source!
```
Comment (pedroslopez) [inline pages/whats-new/2022-01-28-v08.mdx]: note: `And` is used twice here

**Comment by pedroslopez** [comment]
We could probably also link to the blogpost now that it's up https://www.grouparoo.com/blog/v08-release

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**